css/style.css

```diff
@@ -89,6 +87,38 @@ video {
   background-color: #003c8f;
 }
 
+/* Small hero */
+.small-hero__image--quote {
+  background: url("../assets/images/IMG_8453.JPG") no-repeat center;
+}
+.small-hero__image--contact {
+  background: url("../assets/images/IMG_8056.JPG") no-repeat center;
+}
+.small-hero__image--gallery {
+  background: url("../assets/images/IMG_8056.JPG") no-repeat center;
+}
+
+.small-hero {
+  height: 200px;
+  background-size: cover;
+}
+
+.small-hero__layer {
+  height: 100%;
+  background-color: rgba(0, 60, 143, 0.5);
+}
+
+.small-hero__title {
+  font-family: "Roboto-Bold", sans-serif;
+  margin-top: -70px;
+  margin-right: 5%;
+  margin-left: 5%;
```

Comment:
Don't use percentage margins here, the hero title will always be 20px from the left edge. and centered in desktop.

Reply:
Done